### DISCUSSION TITLE
Module is not available

### DISCUSSION THREAD

**Discussion by arishwani** [BishopFox/sliver#769] [General]
WHen I run a module I get response back stating, "Could not load extension: no extension file found for windows/386" 

This error happened for many modules, is this expected or what is issue here? 

Thank you

**Comment by rkervella** [comment]
I assume you're talking about some extensions from the [sliverarmory](https://github.com/sliverarmory). It just means no version was compiled for this platform / architecture couple. What extensions are you trying to run?

**Comment by arishwani** [comment]
Yes, they are extensions from sliver/armory, 
some of the extensions that I get that error for: 
 c2tc-kerberoast
c2tc-lapsdump

**Reply by moloch--** [reply]
The implant is running in a 32-bit process or on a 32-bit system, these BOFs currently do not support 32-bit.

**Comment by arishwani** [comment]
Thank you @moloch--  that make sense. I appreciate the quick reply.
